test/middleware_test.rb

```diff
@@ -78,4 +78,23 @@ def call(env)
     @app.use FreezeMiddleware
     get '/Foo'
   end
+
+  class SpecialConstsMiddleware < MockMiddleware
+    def call(env)
+      req = Rack::Request.new(env)
+      req.update_param('s', :s)
+      req.update_param('i', 1)
+      req.update_param('c', 3.to_c)
```

Comment:
also add `Rational` to this list? `(1/1).to_r`

Reply:
What's the motivation? Fully testing special consts?
If so, it's the same complexity to have that as I mentiond. We need to add more examples such as BigDecimal, Float etc. I think it's not reasonable.
Otherwise, lmk.

Reply:
np, thx